modules/sno-du-configuring-logging-locally-and-forwarding.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-configuring-logging-locally-and-forwarding_{context}"]
+= Configuring logging locally and forwarding
+
+To be able to debug a single node distributed unit (DU), logs need to be stored for further
+analysis.
+
+.Procedure
+
+* Use the following example to configure logging:
++
```

Comment:
. Edit the `ClusterLogging` custom resource (CR) in the `openshift-logging` project:

Reply:
changed.